sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -130,6 +130,47 @@ class ColumnarBatchSuite extends SparkFunSuite {
       }
   }
 
+  testVector("[SPARK35867] Boolean APIs", 1024, BooleanType) {
```

Comment:
nit: don't need to put `[SPARK35867]` here.

Reply:
Updated